pkg/collector/stats/types/types.go

```diff
@@ -130,25 +140,25 @@ func (s *UInt64StatCollection) SetDeltaStat(key string, newDelta uint64) {
 }
 
 // SumAllDeltaValues aggregates the delta metrics of all sources (i.e., stat keys)
-func (s *UInt64StatCollection) SumAllDeltaValues() uint64 {
+func (s UInt64StatCollection) SumAllDeltaValues() uint64 {
 	sum := uint64(0)
```

Comment:
Unrelated, but caught my eye .. wouldn't this overflow as well? 
Shouldn't we use https://pkg.go.dev/math/big here?

Reply:
Tracking this discussion here:
https://github.com/sustainable-computing-io/kepler/issues/1594